lib/routes/controller.js

```diff
@@ -24,6 +27,7 @@ class Controller {
         this.app.post(
```

Comment:
Would it make sense to allow additional content types as an extra argument? And have the default only accept `application/json`?

Reply:
Probably yes.